features/kernel/src/main/resources/features.xml

```diff
@@ -81,4 +82,10 @@
     <feature name="jodah-failsafe" version="${project.version}">
         <bundle>wrap:mvn:net.jodah/failsafe/${jodah-failsafe.version}</bundle>
     </feature>
+
+    <feature name="healthchecks" description="Health and Readiness Checks" version="${fabric8.version}">
```

Comment:
❓ Is there a reason we didn't use their features file? https://github.com/fabric8io/fabric8/blob/master/components/fabric8-karaf/fabric8-karaf-features/src/main/resources/feature.xml

Reply:
I think only because I didn't know if we wanted the other features that were in it

Reply:
It doesn't look like they're adding any additional dependencies in those features that we don't already have. I'd be a fan of having our karaf features file have a repository tag that points to their feature file. One less feature for us to maintain 😄

Reply:
Yeah we can definitely do that, just wasn't sure if we wanted to pull in this stuff as well: https://github.com/fabric8io/kubernetes-client/blob/master/platforms/karaf/features/src/main/resources/feature.xml since it is referenced in that `fabric8-karaf` feature repo there shouldn't be a problem with pulling all that in AFAICT, just wasn't sure if we needed it at the moment